R/GatingSet_Methods.R

```diff
@@ -528,6 +528,9 @@ setReplaceMethod("flowData",signature(x="GatingSet"),function(x,value){
 #' @rdname gs_cyto_data
 #' @export
 setReplaceMethod("gs_cyto_data",signature(x="GatingSet"),function(x,value){
+      if(class(value) != "cytoset") {
```

Comment:
maybe add type checking and throw user-friendly error if it's not flowset

Reply:
Apologies for the delay @mikejiang, I have added additional type checks to ensure `value` is a flowSet or cytoset only. I will submit another PR after this one to fix some other inconsistency issues I have encountered.